### PR TITLE
Update docs and clean up handling of target_ip for tf 0.12

### PR DESCRIPTION
**Pull request by cdaniluk**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -4,7 +4,6 @@ repos:
   rev: v1.19.0
   hooks:
     - id: terraform_fmt
-    - id: terraform_tflint
```
Comment (sblack4) [inline .pre-commit-config.yaml]: why'd you remove this?

File: main.tf [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
-  }
+  dynamic "target_ip" {
+    for_each = var.forward_ips
 
-  target_ip {
-    ip = var.forward_ips[1]
+    content {
+      ip = target_ip.value
+    }
   }
```
Comment (sblack4) [inline main.tf]: How do you feel about this space here? 
```suggestion
```

File: ram.tf [hunk 3]
```diff
@@ -2,7 +2,6 @@ resource "aws_ram_resource_share" "endpoint_share" {
   count                     = length(var.resource_share_accounts) > 0 ? 1 : 0
   name                      = "route53-${var.forward_domain}-share"
   allow_external_principals = false
-  #tags                      = merge(local.common_tags, var.additional_tags)
```
Comment (sblack4) [inline ram.tf]: [It allows tags](https://www.terraform.io/docs/providers/aws/r/ram_resource_share.html). I feel like "tag everything" is a good rule to follow but I also like breaking the rules.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (cdaniluk) [reply .pre-commit-config.yaml]: it breaks commits if you don't have an aws credential

**Pull request merged**